backend/migrations/Version20250917193302.php

```
@@ -0,0 +1,31 @@
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250917193302 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add created_at and updated_at to User entity';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)